### PR TITLE
Make versions_check more reliable in forks

### PR DESCRIPTION
**Pull request by matmair**
This PR:
- Ensures version checks only run on GH actions (the required envs are only available there)
- Adapts the checks to also work on forks

### CODE REVIEW

File: .github/scripts/version_check.py [hunk 1]
```diff
@@ -18,6 +18,9 @@
 
 import requests
 
+REPO = os.getenv('GITHUB_REPOSITORY', 'inventree/inventree')
+GITHUB_API_URL = os.getenv('GITHUB_REPOSITORY', 'https://api.github.com')
```
Comment (SchrodingersGat) [inline .github/scripts/version_check.py:22]: These are both accessing the same environmental variable - is this intentional?

Comment (matmair) [reply .github/scripts/version_check.py:22]: No, that is a typo. Good catch!

**Comment by SchrodingersGat** [comment]
@matmair unfortunately this seems to have broken our docker builds:

image

**Comment by matmair** [comment]
I see the problem, will submit a fix shortly

**Pull request merged**